src/containerapp/azext_containerapp/tests/latest/utils.py

```diff
@@ -234,6 +234,7 @@ def create_and_verify_containerapp_up_for_default_registry_image(
 
         # Assert that the Containre App only has one container and the source to cloud image is used
         app = test_cls.cmd(f"containerapp show -g {resource_group} -n {app_name}").get_output_in_json()
```

Comment:
I test the test `test_containerapp_up_source_with_default_registry_image` you rerun at this PR, it is not set `force_single_container_updates` to true. Please make sure the test you used is meaningful.

`az containerapp up -g arc-appint-forxinyu12-rg0 --environment envforregistry --name xinyutest --source "C:\Users\xinyupang.REDMOND\source\azure-cli-extensions\src\containerapp\azext_containerapp\tests\latest\data\source_built_using_source_to_cloud" --ingress external --target-port 80  --location "North Central US"`